packages/cloud-cognitive/src/components/CreateModal/CreateModal.test.js

```diff
@@ -141,8 +138,60 @@ describe(componentName, () => {
   });
 
   it('applies focus to selected element', () => {
-    const { container } = renderComponent({ primaryFocus });
-    const firstInput = container.querySelector(primaryFocus);
+    const { container } = renderComponent(
+      { selectorPrimaryFocus },
+      <TextInput
+        key="form-field-1"
+        id="1"
+        labelText="Text input label"
+        helperText="Helper text goes here"
+        placeholder="Placeholder"
+      />
+    );
+    const firstInput = container.querySelector(selectorPrimaryFocus);
     expect(firstInput === document.activeElement).toBeTruthy();
   });
+
+  it('throws an error if there are more than 4 child nodes inside of the modal', () => {
+    const { container } = renderComponent({}, [
+      <TextInput
+        key="form-field-1"
+        id="1"
+        labelText="Text input label"
+        helperText="Helper text goes here"
+        placeholder="Placeholder"
+      />,
+      <TextInput
+        key="form-field-2"
+        id="2"
+        labelText="Text input label"
+        helperText="Helper text goes here"
+        placeholder="Placeholder"
+      />,
+      <TextInput
+        key="form-field-3"
+        id="3"
+        labelText="Text input label"
+        helperText="Helper text goes here"
+        placeholder="Placeholder"
+      />,
+      <TextInput
+        key="form-field-4"
+        id="4"
+        labelText="Text input label"
+        helperText="Helper text goes here"
+        placeholder="Placeholder"
+      />,
+      <TextInput
+        key="form-field-5"
+        id="5"
+        labelText="Text input label"
+        helperText="Helper text goes here"
+        placeholder="Placeholder"
+      />,
+    ]);
+    expect(() => {
+      render(...container);
+    }).toThrow();
```

Comment:
And a bit of optional clean-up.

```suggestion
    }).toThrow();

    error.mockRestore();
```